website/www/site/assets/scss/_navbar-desktop.scss

```diff
@@ -33,7 +33,8 @@
    height: 96px;
    width: 100%;
    align-items: center;
-   margin-bottom: 30px;
+   //usually 30px, change for 0 until summit is over and banner is removed
```

Comment:
same here, let's format this comment as a todo to make it easier to find later:

// TODO: revert to 30px after summit is over